SECURITY.md

```diff
@@ -0,0 +1,35 @@
+# Security Policy
+
+## Supported Versions
+
+Redis is generally backwards compatible with very few exceptions, so we
+recommend users to always use the latest version to experience stability,
+performance and security.
+
+We generally backport security issues to a single previous major version,
+unless this is not possible or feasible with a reasonable effort.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 2.8.x   | :white_check_mark: |
+| 2.6.x   | :white_check_mark: |
+| 2.4.x   | :white_check_mark: |
```

Comment:
2.4 is EOL. Please remove this line and change the next one to "<2.6"

Reply:
Solved